.github/CODEOWNERS

```diff
@@ -80,6 +82,4 @@
 /runtime/src/iree/hal/vulkan/ @antiagainst @ScottTodd
 
 # Other IREE directories
-/samples/ @ScottTodd
 /iree/test/ @ghost
```

Comment:
GitHub has started complaining about `@ghost`. That's annoying

Reply:
I think it's been complaining about it since you started using it :P